pkg/cmd/ssh/connect_information.go

```diff
@@ -198,6 +198,7 @@ func (p *ConnectInformation) String() string {
 		p.Bastion.UserKnownHostsFiles,
 		nodeHostname,
 		p.NodePrivateKeyFiles,
+		"",
```

Comment:
adapt `ConnectInformation` struct to have a `User` field. 

```suggestion
		p.User,
```

Reply:
a071555